x-pack/plugins/maps/public/legacy_visualizations/tile_map/tile_map_renderer.tsx

```diff
@@ -5,16 +5,19 @@
  * 2.0.
  */
 
-import React, { lazy } from 'react';
+import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import type { ExpressionRenderDefinition } from '@kbn/expressions-plugin/common';
+import { dynamic } from '@kbn/shared-ux-utility';
 import type { TileMapVisRenderValue } from './tile_map_fn';
-import { LazyWrapper } from '../../lazy_wrapper';
 import { TILE_MAP_RENDER } from './types';
 
-const getLazyComponent = () => {
-  return lazy(() => import('./tile_map_visualization'));
-};
+const Component = dynamic(async () => {
```

Comment:
Oh neat. So using `dynamic` avoids needing to specify `// eslint-disable-next-line import/no-default-export`?

Reply:
not quite. returning `{ default }` prevents needing a component to export as default